app/decorators/service_provider_session_decorator.rb

```diff
@@ -129,15 +115,16 @@ def failure_to_proof_url
   end
 
   def sp_alert?(path)
-    custom_alert? ? alert_not_excluded_for_path?(path) : default_alert?
-  end
-
-  def sp_alert_name
-    CUSTOM_SP_ALERTS.dig(sp_name, :i18n_name)
-  end
-
-  def sp_alert_learn_more
-    custom_alert? ? CUSTOM_SP_ALERTS.dig(sp_name, :learn_more) : 'https://login.gov/help/'
+    path_to_section_map = { '/': 'sign_in',
```

Comment:
Is there a reason not to use the path helpers here?

Reply:
Nope, only reason is that I didn't realize I could pass in the locale as an option to the path helper until Doug pointed it out in his comment below!! 😆